.github/workflows/build.yml

```diff
@@ -118,6 +122,29 @@ jobs:
       cmake_preset_type: ${{needs.common_config.outputs.cmake_preset_type_resolved}}
       matrix: ${{needs.common_config.outputs.linux_matrix}}
 
+  linux-compat-tests:
+    needs: [follower-linux]
```

Comment:
Have to do it in parallel with the other tests in `build_steps.yml` itself. Otherwise, it will make each build even longer.